src/styles/authBlueSide.scss

```diff
@@ -0,0 +1,45 @@
+$main_color:#243665 ;  
```

Comment:
The primary color is also already declared in our base.scss file [here](https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/blob/ft-login-page-185218916/src/styles/base.scss#L4)